.gitignore

```diff
@@ -6,3 +6,5 @@ project/target
 .virtualenv
 __pycache__
 *.pyc
+.bloop
+.metals
```

Comment:
Please add a new line to get rid of this.

Also, you are using metals?  Haven't even tried it yet (or haven't tried it in a while).  Wonder how it is going.

Reply:
I always forget about adding a new line at the end of the files.

I use vscode for haskell and sometimes for scala. Metals is great but some features from idea are not yet there. Metals has a wonderful type checking (except for kind-projector that is not working) which Intellij lacks.